src/components/floating-menu/floating-menu.js

```diff
@@ -169,6 +183,7 @@ class FloatingMenu extends mixin(createComponent, eventedShowHideState) {
       }
       this._getContainer().appendChild(this.element);
       this._place();
+      (this.element.querySelector(this.options.selectorPrimaryFocus) || this.element).focus();
```

Comment:
@asudoh My concern is that if a dev is dynamically loading or changing the menu items and doesn't properly add `selectorPrimaryFocus` then the end user's experience will be extremely broken. When I was researching this I had come across that stackoverflow post you mentioned so I definitely understand that  "first focusable element" is tricky. Maybe a better addition would be if `selectorPrimaryFocus` isn't found try and find the first `a`, `button`, `input`, `select`, or `[tabindex]`. That won't cover all focusable elements but it will cover the vast majority of use cases.